packages/patternfly-4/react-inline-edit-extension/src/components/InlineEdit/editableRowWrapper.d.ts

```diff
@@ -15,8 +15,7 @@ export interface EditableRowWrapperProps extends RowWrapper {
   row: EditableRowWrapperRow;
 }
 
-export interface EditableRowWrapper {
-  (bodyComponent: Component): FunctionComponent<EditableRowWrapperProps>;
-}
+export type EditableRowWrapper = (bodyComponent: Component) => FunctionComponent<EditableRowWrapperProps>;
```

Comment:
This is a breaking change. Changing from 'interface' to 'type'

Reply:
Reverted